src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        /*
+         * String logo = " ____        _        \n"
+         * + "|  _ \\ _   _| | _____ \n"
+         * + "| | | | | | | |/ / _ \\\n"
+         * + "| |_| | |_| |   <  __/\n"
+         * + "|____/ \\__,_|_|\\_\\___|\n";
+         */
+        String logo = " ____   _              __    __ \n"
+                + "/  __\\ | |     _____  |   \\/   |\n"
+                + "| /    | |    /  _  \\ | |\\  /| |\n"
+                + "| \\ __ | |___ | |_| | | | \\/ | |\n"
+                + "\\____/ |____/ \\_____/ |_|    |_|\n";
+
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm CLoM!\n");
+        System.out.println(" What can I do for you?\n");
+
+        int counter = 0; // number of items in the list
```

Comment:
Name is too generic, something more specific like `xCounter` where x is line or input or task depending on your use case would work better